README.md

```diff
@@ -271,6 +271,12 @@ Set the schema
 
 The Default value is 'public'
 
+##### `$oracle_sid`
```

Comment:
Could we change this parameter name?  `oracle_sid` doesn't sound like a Boolean option.  Maybe `oracle_use_sid`??

Reply:
Naming things is hard! ;)

Reply:
I renamed it to the suggested param.